src/main/java/core/basesyntax/serviceimpl/ExecuteOperations.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.strategyimpl.OperationStrategyImpl;
+import java.util.List;
+
+public class ExecuteOperations {
```

Comment:
this is a verb, class names should be nouns
```suggestion
public class TransactionExecutor {
```